capture/SharpCap/samples/Totality.py

```diff
@@ -0,0 +1,129 @@
+# Capture of different exposures for number of sets of exposures,
+#                                    time of run
+#                                    or continuous loop of sets
+# Camera in Still Mode for captures
+# resets to Live mode with original exposure/gain/resolution after run
+#           
+# Authors:  Chris Mandrell
+#           Matt Penn
+# Written: 3/20/2023
+# Last Edit: 4/4/2023 CM
+#####################################################################################
+
+import time
+#from datetime import datetime
+
+#date_form = datetime.now().strftime("%d_%b_%Y") 
+
+### List of variables to be changed by user (User Input) #########################################
+path = r'C:\DEB\Exmouth\NoFilter\Totality' # path for folder
+#run_name = "bob"
+
+between_exp = 0.0 # time between each exposure (seconds)
+between_set = 0.0 # time between sets of exposures in exposure List (seconds)
+
+exposure = ( 0.4, 4.0, 40.0, 400.0, 4000.0 ) # exposure tuple (ms)
+
+Time = True # Allow capture control by time
```

Comment:
Let's not use capitalized names for flags, as the convention is to use this for classes.

I'd suggest "use_duration"  and "use_maxcount" , and rename number to maxcount.

and suppress_settings_file for Suppress.